apps/services/auth/admin-api/src/app/v2/scopes/test/me-scopes.spec.ts

```diff
@@ -265,21 +260,42 @@ const createTestCases: Record<string, CreateTestCase> = {
       body: expectedCreateOutput,
     },
   },
+  'should publish scope and have access as current user': {
```

Comment:
Maybe rephrase to explain expanded create/post functionality, since the rest API has no concept of publish.